job-server/src/main/scala/spark/jobserver/util/HealthCheck.scala

```diff
@@ -21,26 +23,27 @@ trait HealthCheck {
 
 }
 
-class ActorsHealthCheck(supervisor: ActorRef, jobInfo: ActorRef, daoActor: ActorRef)
+class ActorsHealthCheck(supervisor: ActorRef, daoActor: ActorRef)
     extends HealthCheck {
 
     val logger = LoggerFactory.getLogger(getClass)
 
     def isHealthy() : Boolean = {
       logger.info("Receiving healthz check request")
       import spark.jobserver.ContextSupervisor._
-      import spark.jobserver.JobManagerActor._
-      import spark.jobserver.JobInfoActor._
       import JobDAOActor._
       import spark.jobserver.CommonMessages._
       var actorsAlive: Int = 0
       implicit val duration: Timeout = 30 seconds
       val supervisorFuture = (supervisor ? GetContext("dummycontext")).mapTo[NoSuchContext.type]
-      val jobInfofuture = (jobInfo ? GetJobStatus("dummyjobid")).mapTo[NoSuchJobId.type]
-      val jobDaofuture = (daoActor ? GetJobInfos(1)).mapTo[JobInfos]
-      val jobResultfuture = (jobInfo ? GetJobResult("dummyjobid")).mapTo[NoSuchJobId.type]
+      val jobDaofuture = (daoActor ? GetJobInfos(1, None)).mapTo[JobInfos]
```

Comment:
Could we capitalize future -> Future everywhere?  Just to stick with camelCase

Reply:
Done